docker-compose.yml

```diff
@@ -20,7 +20,8 @@ x-env-mapping: &env
 services:
   worker: &worker
     <<: *env
-    image: addons/addons-server:latest
+    # image: addons/addons-server:latest
+    build: .
```

Comment:
mmm I assume you had that for local testing ? Once we build a new image and push it it shouldn't be necessary, so I vote for not keeping it in the PR

Reply:
Indeed, sorry I forgot to roll that change back. Fixed in 4f09ec7d39